src/WebJobs.Script.WebHost/Diagnostics/LinuxAppServiceEventGenerator.cs

```diff
@@ -69,8 +74,17 @@ private static void WriteEvent(LinuxAppServiceFileLogger logger, string evt)
             logger.Log(evt);
         }
 
+        private void ConsoleWriter(string evt)
```

Comment:
nit: method name -> `WriteEvent`?

Argument name to something fully spelled out (e.g. eventPayload, etc.)